ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/target/P2TargetUtils.java

```diff
@@ -1418,7 +1420,16 @@ private Collection<URI> getArtifactRepositories(ITargetDefinition target) throws
 			// get all the artifact repos we know in the manager currently
 			result.addAll(Arrays.asList(manager.getKnownRepositories(IRepositoryManager.REPOSITORIES_ALL)));
 
-			// Add in the IDE profile bundle pool and all known workspaces
+			if ("true".equals(System.getProperty("pde.usePoolsInfo", "true"))) { //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```

Comment:
```suggestion
			if (Boolean.parse(System.getProperty("pde.usePoolsInfo", "true"))) { //$NON-NLS-1$ //$NON-NLS-2$
```
Would be a bit more flexible since `TRUE` would also be ok and is IMHO a bit more readable.

Reply:
Done.